src/material-experimental/mdc-checkbox/checkbox.ts

```diff
@@ -211,7 +210,7 @@ export class MatCheckbox implements AfterViewInit, OnDestroy, ControlValueAccess
   private _checkboxAdapter: MDCCheckboxAdapter = {
     addClass: (className) => this._setClass(className, true),
     removeClass: (className) => this._setClass(className, false),
-    forceLayout: () => this._platform.isBrowser && this._checkbox.nativeElement.offsetWidth,
+    forceLayout: () => this._checkbox.nativeElement.offsetWidth || 0,
```

Comment:
The `|| 0` technically isn't necessary here, but I added it in case something was depending on the return value of this function.